gui/drawable/inc/drawable.hpp

```diff
@@ -29,17 +27,18 @@ class Drawable {
     const std::unordered_map<std::shared_ptr<Drawable>, Drawable::Position> &getChildDrawables()
         const;
     void addChild(const std::shared_ptr<Drawable> &child, Position p);
+    void removeChild(const std::shared_ptr<Drawable> &child);
+    void clearChildren();
+    void setColor(const Color &c) { color = c; }
 
    protected:
     explicit Drawable() = default;
 
-    // TODO change in the drawing engine PR
-    // NOLINTBEGIN
-    Color color = {max_color_val, max_color_val, max_color_val, max_color_val};
+    // NOLINTNEXTLINE
     std::list<Position> shape;
```

Comment:
what is wrong with this line - why does it need NOLINTNEXTLINE

Reply:
It is a protected variable inside a class, only private members are allowed

Reply:
[cppcoreguidelines-non-private-member-variables-in-classes](https://clang.llvm.org/extra/clang-tidy/checks/cppcoreguidelines/non-private-member-variables-in-classes.html)